src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersionImport.cs

```diff
@@ -8,11 +8,15 @@ public class DataSetVersionImport : ICreatedUpdatedTimestamps<DateTimeOffset, Da
 {
     public Guid Id { get; init; }
 
-    public required Guid DataSetVersionId { get; set; }
+    public Guid DataSetVersionId { get; set; }
```

Comment:
I can reinstate this but it means we can't set up a data set version with an import and save them both together like this

```csharp
        var dataSetVersion = new DataSetVersion
        {
            ...
            Imports =
            [
                new DataSetVersionImport
                {
                    ...
                }
            ]
        };
```

Reply:
Yes it would fail to save in the database due to the foreign key constraint.

The database foreign key column `DataSetVersionImport.DataSetVersionId` is not nullable due to the type being `Guid` rather than `Guid?` and removing the `required` keyword by changing `public required Guid DataSetVersionId { get; set; }` to `public Guid DataSetVersionId { get; set; }` in `DataSetVersionImport` wouldn't alter that.

I think the combined save approach is nice but to keep things consistent for now I've reverted this change by adding the `required` keyword back in and made a change to save the `DataSetVersionImport` separately.